lmod_proxy/web.py

```diff
@@ -53,3 +57,22 @@ def index(user):
         Flask.response
     """
     return redirect(url_for('edx_grades.index'))
+
+
+@app.route('/status', methods=['GET'])
+def status():
+    """Route to get app cert expiration date
+
+    Return: json object containing app_cert_expiratin date and status
+    """
+    app_cert_file = open(LMODP_CERT, 'rt').read()
```

Comment:
nit: I would change the variable name to reflect the fact that it's actually storing the contents of the file and is not a pointer to the file itself.